### ISSUE TITLE
Make it clear this is not an exact implementation of DEPTA (and MDR-2)

### ISSUE THREAD

**Issue by kmike** [pydepta/pydepta#10]
I've read "Web Data Extraction Based on Partial Tree Alignment" paper, and it says that DEPTA uses MDR-2 data regions extraction algorithm, and that MDR-2 differs from MDR in how it constructs trees from HTML data: instead of relying on tag hierarchy it renders tree in a browser and uses bounding boxes of elements to create a tree. Also, visual information is used to compute better data regions: gap between two data records in a data region must be no smaller than any gap within a data record. I don't think pydepta implements any of this; the implementation looks more like MDR. 

Maybe add a note to README that this is not a "real" DEPTA, but more like MDR + "Data extraction" part from DEPTA?

**Comment by tpeng** [comment]
yes. pydepta is MDR with partial tree alignment. i will add a note
